.ghal.rules.json

```diff
@@ -205,6 +235,117 @@
               "(?i).*docs\/csharp.*": {
                 "labels-add": ":books: Area - C# Guide"
               },
+              "(?i).*docs\/csharp\/misc.*": {
+                "labels-add": ":card_file_box: Technology - C# / VB diagnostics"
+              },
+              "(?i).*docs\/csharp\/whats-new.*": {
+                "labels-add": ":card_file_box: Technology -  C# What's New"
+              },
+              "(?i).*docs\/csharp\/how-to.*": {
+                "labels-add": ":card_file_box: Technology -  C# Fundamentals"
+              },
+              "(?i).*docs\/csharp\/linq.*": {
+                "labels-add": ":card_file_box: Technology -  LINQ"
+              },
+              "(?i).*docs\/csharp\/programming-guide\/main-and-command-args.*": {
+                "labels-add": ":card_file_box: Technology -  C# Fundamentals"
+              },
+              "(?i).*docs\/csharp\/programming-guide\/indexers.*": {
+                "labels-add": ":card_file_box: Technology -  C# Fundamentals"
+              },
+              "(?i).*docs\/csharp\/programming-guide\/generics.*": {
+                "labels-add": ":card_file_box: Technology -  C# Fundamentals"
+              },
+              "(?i).*docs\/csharp\/programming-guide\/strings.*": {
+                "labels-add": ":card_file_box: Technology -  C# Fundamentals"
+              },
+              "(?i).*docs\/csharp\/programming-guide\/types.*": {
+                "labels-add": ":card_file_box: Technology -  C# Fundamentals"
+              },
+              "(?i).*docs\/csharp\/programming-guide\/statements-expressions-operators.*": {
+                "labels-add": ":card_file_box: Technology -  C# Fundamentals"
+              },
+              "(?i).*docs\/csharp\/programming-guide\/interop.*": {
+                "labels-add": ":card_file_box: Technology -  C# Advanced concepts"
+              },
+              "(?i).*docs\/csharp\/programming-guide\/unsafe-code-pointers.*": {
+                "labels-add": ":card_file_box: Technology -  C# Advanced concepts"
```

Comment:
```suggestion
                "labels-add": ":card_file_box: Technology -  C# Advanced Concepts"
```

Reply:
This one is correct today, matching the label text.